### ISSUE TITLE
Carousel is not keyboard accessible

### ISSUE THREAD

**Issue by corriespondent** [sahilsaha7773/react-carousel-minimal#11]
The carousel is not keyboard accessible. You cannot select the arrows or thumbnails when you tab through the page. It would be nice to be able to use left/right arrow keys when you have tabbed into the carousel or the thumbnails, as well.

Changing the next/prev to <button> or adding tabIndex attribute to the next/prev/dots/thumbnails would allow for keyboard access at the minimum.
